rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/RetreatTurn.java

```diff
@@ -1,18 +1,17 @@
 package pl.oaza.warszawa.dor.rekolekcje.api.participants.domain;
 
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.NoArgsConstructor;
-import lombok.Setter;
-import pl.oaza.warszawa.dor.rekolekcje.api.participants.value.RetreatTurnValue;
-
 import javax.persistence.Entity;
 import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.JoinColumn;
 import javax.persistence.ManyToOne;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
```

Comment:
\[Checkstyle\] WARNING: Import statement for 'lombok\.Builder' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.